src/helpers/stateSnapshot.ts

```diff
@@ -20,20 +19,16 @@ export const stateSnapshotter = errorHandler(_stateSnapshotter)
 async function _stateSnapshotter(
```

Comment:
Would be great to create two pre handlers for this method:

`subStateSnapshotter(block: SubstrateBlock)`

And 

`evmStateSnapshotter(block: EvmBlock)`

Which do the unpacking and then call the chain independent `stateSnapshotter(block, timestamp)`

What do you think?